### DISCUSSION TITLE
What is the ubuntu login?

### DISCUSSION THREAD

**Discussion by noelfranthomas** [cirruslabs/tart#708] [Q&A] [answered]
I'm trying to run `ghcr.io/cirruslabs/ubuntu:latest`, but when I run it the program takes me to a login screen. What should the username and password be for this?

Thanks for any help in advance!

**Comment by edigaryev** [comment, marked as answer]
Hello Noel 👋 

It's `admin`/`admin`, the same is for macOS images.

Looks like we should document this better on the website 🤔  This is currently only mentioned in [SSH access](https://tart.run/quick-start/#ssh-access) section.

**Reply by DrGo** [reply]
I think that will be appreciated

**Reply by edigaryev** [reply]
Done in https://github.com/cirruslabs/tart/pull/709, the changes should be live shortly 🚀

**Reply by DrGo** [reply]
Thanks for the quick reply

**Reply by noelfranthomas** [reply]
Thank you!
